LibVLCSharp/Platforms/UWP/VideoView.cs

```diff
@@ -0,0 +1,233 @@
+﻿namespace LibVLCSharp.Platforms.UWP
+{
+    using LibVLCSharp.Shared;
+    using SharpDX;
+    using SharpDX.Direct3D11;
+    using SharpDX.DXGI;
+    using SharpDX.Mathematics.Interop;
+    using System;
+    using System.Runtime.InteropServices;
+    using Windows.UI.Xaml.Controls;
+    using Device3 = SharpDX.DXGI.Device3;
+
+    /// <summary>
+    /// The VideoView implementation for UWP applications
+    /// </summary>
+    public class VideoView : UserControl, IVideoView
```

Comment:
Should this inherit from `SwapChainPanel`?

Reply:
The panel is inside here, not sure if we should change that and make it a panel itself. a user control would be more flexible for the future